Gemfile

```diff
@@ -75,23 +75,22 @@ gem 'rest-client'
 gem 'noid', '~> 0.8'
 
 group :test do
-  gem "rspec-its"
```

Comment:
We were using rspec-its to backport rspec's `its {}` functionality which was removed from rspec. We were using this in three places. I have refactored these so we no longer need this. `its {}` syntax does lots of magic and isn't intuitive to understand. Reason it has been removed from rspec. More info about why it was removed here:
https://gist.github.com/myronmarston/4503509